### ISSUE TITLE
Ability To Add Templated and Also Custom Notes Information Associated With An Image - Print Sizes Available (etc)

### ISSUE THREAD

**Issue by ildyria** [LycheeOrg/Lychee#2312]
### Discussed in https://github.com/LycheeOrg/Lychee/discussions/2305

<div type='discussions-op-text'>

<sup>Originally posted by **vepicurean** February 28, 2024</sup>
### Enhancement

Maybe I'm missing if Lychee out of the box has the ability to add templated or custom user information for an uploaded image or album. I've looked, and just can seem to see where this functionality would reside and how it would look. 

I would like to ability to include user information on available print sizes for an image or album ( etc 12in x 18in, 36in x 42in, that I can define the sizes available). Edition information by print size. 12in x 18in - edition of 9, 36in x 42in - edition of 6</div>

> Also what you want is not Custom Notes, but more like an Edition management.
What you just requested could be easily added manually in the description section of the picture.

### TL;DR

Add possible edition management module, e.g. number of prints per size, availability left. Could also be used for pricing and different sizes (medium vs big) etc.

**Comment by ildyria** [comment]
#4388
